test.js

```diff
@@ -0,0 +1,66 @@
+// 1. Programming Basics
+// 'Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'
+// Create an array of the dishes listed above. Loop through the array using aforloop.
+// 1.Log out all the dishes.
+// 2.Log out a randomly chosen dish (randomly generated  index).
+
+const dishes = [
+  "Spaghetti",
+  "Lasagne",
+  "Beans on toast",
+  "Porridge",
+  "Tomato soup",
+  "Salmon",
+];
+function getDishes() {
+  for (i = 0; i < dishes.length; i++) {
+    console.log(dishes[i]);
+  }
+}
+getDishes();
+
+// 2. DOM manipulation
+// Using JavaScript, create a button and add it to the html (the document)
+// When the button is clicked:
+// -Insert an h1 tag below the button with the text “Javascripttest”.
+// -Change the background color of the document body.
+
+const testBtn = document.createElement("button");
+testBtn.innerHTML = "Test Button";
+document.body.appendChild(testBtn);
+testBtn.addEventListener("click", function () {
+  const h1 = document.createElement("h1");
+  h1.innerHTML = "Javascripttest";
+  document.body.style.backgroundColor = "orangered";
+  document.body.after(h1);
+});
+
```

Comment:
Nice! :)
A possible reconsideration would be to not create and populate the inner HTML of a new header on every click.
You could have it ready outside of the event listener callback and just use it when needed, as it is an element that does not change and therefore there is no need to load that part of the process onto the event callback.

Reply:
Hi shpomp, thanks for the review.
I made some of the recommended changes, but I did not quite understand if what you suggested here is what I did.

Reply:
No, sorry if I made it confusing by how I worded it 🙏🏽 
I meant that these are inside the onClick callback:
```
 const h1 = document.createElement("h1");
  h1.innerHTML = "Javascripttest";
```
So `h1` is created and the inner HMTL of it is filled with every click. 
But it is the same every time, right? 
So it could just live outside of the function:
```
    ...
    const h1 = document.createElement("h1");
    h1.innerHTML = "Javascripttest";

testBtn.addEventListener(
  "click",
  () => {
    document.body.style.backgroundColor = "orangered";
    ...
```

Reply:
Oh yes, 
as simple as that. 😊
I understand now what you were saying, so if I need the h1 tag again I don't need to declare it once more, of course 🤩
Thanks...